### DISCUSSION TITLE
求助，最新版本的模版都无法导入

### DISCUSSION THREAD

**Discussion by keji1115** [windingwind/zotero-better-notes#500] [Q&A] [answered]
Template [item]MeteData table Error: ReferenceError: topItem is not defined
试了几个笔记显示的都是这个，请问怎么样才可以解决？非常感激且期待您的回复！

**Comment by xinchen0430** [comment]
我也遇到了一样的问题，同问

**Reply by Anphetamine** [reply]
模板开头是不是要加上 [Item]

**Reply by anandkumar89** [reply]
I'm also facing same problem. Where can I find a starter template which works simply by pasting in template window. I can improve from there. 
<img width="479" alt="Screenshot 2023-06-19 at 2 19 14 AM" src="https://github.com/windingwind/zotero-better-notes/assets/12571250/1a3b25c0-0a97-4e2e-b3f2-011799d6c376">


Here's a screenshot of template I tried from discussion.

<img width="790" alt="Screenshot 2023-06-19 at 2 15 14 AM" src="https://github.com/windingwind/zotero-better-notes/assets/12571250/96f16dda-3742-4667-853c-ac9b8491ab19">

**Reply by Eppendora** [reply, marked as answer]
Template code has no problem. The only thing to do is that rename the **Template Name**, start with [Item]. i.e.[Item]Your Template Name. This correct name tell the plugin to get the item information. All the best.

**Reply by windingwind** [reply]
@anandkumar89 
And the text in your screenshot is not template CONTENT, but the template SHARE CODE, which is for sharing and is expected to be used by the menu-Edit-Import template from the clipboard, not directly pasted into the template editor.
